openvpn/config/generic_config.go

```diff
@@ -118,3 +121,17 @@ func (c *GenericConfig) SetPersistTun() {
 func (c *GenericConfig) SetPersistKey() {
 	c.SetFlag("persist-key")
 }
+
+// SetScriptParam adds parameter with special handling of the value - it's treated as script name inside script search directory
+// additional needDoubleQuotes param indicates if scriptName should be surounded by double quotes ( example --up --down scripts)
+func (c *GenericConfig) SetScriptParam(paramName, scriptName string, needDoubleQuotes bool) {
```

Comment:
Could be implemented as `optionScript` same as `optionFile` which was created exactly for serialisation problem